File: Samples/C#.Net/CSharpSDKTest/ISmpFileMenuOperationsDlg.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using SalesMatePlusLib;
namespace CSharpSDKTest
{
    public partial class ISmpFileMenuOperationsDlg : Form
    {
        public ISmpFileMenuOperationsDlg()
        {
            InitializeComponent();
        }

        private void BackupDatabtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeBackupDataDlg();
        }

        private void ChangePasswordbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeChangePasswordDlg();
        }

        private void ExportReportbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeExportReportDlg();
        }

        private void Loginbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeLoginDlg();
        }

        private void Logoutbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeLogout();
        }

        private void Exitbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeExit();

            Application.Exit();
        }

        
        private void Printbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokePrintDlg();
        }

        private void PrintOldReceiptbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokePrintOldReceiptDlg();
        }

        private void PrintPreviewbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokePrintPreviewDlg();
        }

        private void PrintSetupbtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokePrintSetupDlg();
        }

        private void RestoreDatabtn_Click(object sender, EventArgs e)
        {
            ISmpFileMenuOperations m_SmpFileMenuOperations;

            m_SmpFileMenuOperations = new SmpFileMenuOperationsClass();

            m_SmpFileMenuOperations.InvokeRestoreDataDlg();
        }
    }
}

File: Samples/VB.Net/VBDotNet/ISmpFileMenuOperationsDlg.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports SalesMatePlusLib
Namespace CSharpSDKTest
	Partial Public Class ISmpFileMenuOperationsDlg
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub BackupDatabtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles BackupDatabtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeBackupDataDlg()
		End Sub

		Private Sub ChangePasswordbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles ChangePasswordbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeChangePasswordDlg()
		End Sub

		Private Sub ExportReportbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles ExportReportbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeExportReportDlg()
		End Sub

		Private Sub Loginbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles Loginbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeLoginDlg()
		End Sub

		Private Sub Logoutbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles Logoutbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeLogout()
		End Sub

		Private Sub Exitbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles Exitbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeExit()

			Application.Exit()
		End Sub


		Private Sub Printbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles Printbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokePrintDlg()
		End Sub

		Private Sub PrintOldReceiptbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles PrintOldReceiptbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokePrintOldReceiptDlg()
		End Sub

		Private Sub PrintPreviewbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles PrintPreviewbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokePrintPreviewDlg()
		End Sub

		Private Sub PrintSetupbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles PrintSetupbtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokePrintSetupDlg()
		End Sub

		Private Sub RestoreDatabtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles RestoreDatabtn.Click
			Dim m_SmpFileMenuOperations As ISmpFileMenuOperations

			m_SmpFileMenuOperations = New SmpFileMenuOperationsClass()

			m_SmpFileMenuOperations.InvokeRestoreDataDlg()
		End Sub
	End Class
End Namespace